### PR TITLE
Optimize reserving CI nodes for debugging

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

**Comment by kbsingh** [comment]
note that as mentioned in communication to ci members, you cant currently fail nodes - this restrictions is going to stay in place for the next week or so, track progress / changes on the ci users list

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
#dotests-debug

**Comment by rtnpro** [comment]
CI is green.

@bamachrn @kbsingh, could you merge it?

File: ci/job.yml [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
         - shell: |
             python ./ci/cccp_ci.py
 
 - builder:
     name: centos-container-pipeline-service-ci-cleanup-builders
     builders:
-        - shell:
-            python ./ci/cccp_ci_cleanup.py
+        - python:
+            !include-raw: ./cccp_ci_cleanup.py
```
Comment (mohammedzee1000) [inline ci/job.yml]: I thought we are not for  !include-raw, so what gives

**Comment by bamachrn** [comment]
LGTM merging

**Pull request merged**